### PR TITLE
Feature change reward ratio

### PR DESCRIPTION
**Pull request by me-vadim**
Change torewards distribution ratio from 70:20:10 to 60:30:10
Add new table to hold torewards distribution ratio

### CODE REVIEW

**Review by AndriiAndrusyshyn — APPROVED** [review]

File: contracts/contracts/rem.system/src/rem.system.cpp [hunk 1]
```diff
@@ -71,6 +80,23 @@ namespace eosiosystem {
       _global2.set( _gstate2, _self );
       _global3.set( _gstate3, _self );
       _global4.set( _gstate4, _self );
+      _global5.set( _gstate5, _self );
+   }
+
+   void system_contract::setstakeshare( double share ) {
```
Comment (roman-tik) [inline contracts/contracts/rem.system/src/rem.system.cpp]: let's combine into one method that sets stake+vote+savings, so we can add a check that total share equals 1

File: contracts/contracts/rem.system/include/rem.system/rem.system.hpp [hunk 2]
```diff
@@ -205,6 +205,17 @@ namespace eosiosystem {
       EOSLIB_SERIALIZE( eosio_global_state4, (continuous_rate)(inflation_pay_factor)(votepay_factor) )
    };
 
+   /**
+    * Defines new global state parameters to store torewards distribution
+    */
+   struct [[eosio::table("global5"), eosio::contract("rem.system")]] eosio_global_state5 {
```
Comment (roman-tik) [inline contracts/contracts/rem.system/include/rem.system/rem.system.hpp]: let's take the opportunity to name our table something like _globalrem1_ or something like this

File: contracts/contracts/rem.system/include/rem.system/rem.system.hpp [hunk 3]
```diff
@@ -540,10 +553,12 @@ namespace eosiosystem {
          global_state2_singleton _global2;
          global_state3_singleton _global3;
          global_state4_singleton _global4;
+         global_state5_singleton _global5;
          eosio_global_state      _gstate;
          eosio_global_state2     _gstate2;
          eosio_global_state3     _gstate3;
          eosio_global_state4     _gstate4;
+         eosio_global_state5     _gstate5;
          rammarket               _rammarket;
```
Comment (roman-tik) [inline contracts/contracts/rem.system/include/rem.system/rem.system.hpp:562]: please start a separate PR to clean up legacy variables like this. (+ legacy actions and commented code)

**Review by roman-tik — CHANGES_REQUESTED** [review]

File: contracts/contracts/rem.system/include/rem.system/rem.system.hpp (hunk 3 above)
Comment (me-vadim) [reply contracts/contracts/rem.system/include/rem.system/rem.system.hpp:562]: AFAIK _rammarket is not a legacy variable

**Review by mark-tsyrulnyk — APPROVED** [review]

**Pull request merged**